### DISCUSSION TITLE
Help me please - ERR13

### DISCUSSION THREAD

**Discussion by Malakay-666** [Kozea/Radicale#1358] [Q&A general]
Hi,
I had to reinstall raspi os because of failing micro SD card. I was using Radicale and PiHole on it. This weekend I bought new card, imaged Raspi OS 64bit with no desktop. Then installed Radicale and set up config and users file (just one user, one password). I am using it just on my LAN, so I set up htpasswd auth with plain option. When I went to web-admin interface and try to login, it told me 500 internal server error, login and password text field disappeared and only black screen appeared with some message like "contact administrator" or something. When I run "python -m radicale" with debug logging I found this : 
PermissionError: [Errno 13] Permission denied: '/var/lib/radicale/collections/.Radicale.lock' radicale

What does this mean and how can I fix it? 

Thanks for the help.

**Comment by Malakay-666** [comment]
Its interesting that last time I installed it few months ago, I did it on first attempt and it was working flawlessly. Maybe you can provide some step-by-step guide? :)

**Comment by GreenGusher** [comment]
what is the variable "filesystem_folder"? 
Do you have file permissions for it as a user?

"""
Storage
Data is stored in the folder /var/lib/radicale/collections. The path can be changed with the following configuration:

[storage]
filesystem_folder = /path/to/storage
Security: The storage folder should not be readable by unauthorized users. Otherwise, they can read the calendar data and lock the storage. You can find OS dependent instructions in the [Running as a service](https://radicale.org/master.html#running-as-a-service) section.
"""

or 

Are you running a service as user radicale"

"""
Linux with systemd system-wide
Create the radicale user and group for the Radicale service. (Run useradd --system --user-group --home-dir / --shell /sbin/nologin radicale as root.) The storage folder must be writable by radicale. (Run mkdir -p /var/lib/radicale/collections && chown -R radicale:radicale /var/lib/radicale/collections as root.)
"""

**Comment by Malakay-666** [comment]
I am not using it as service at the moment, because I freshly installed it. I think i used it under "pi" user, but  i am not sure, i cant remember last setup. I think i didnt ever create other user than pi.

**Comment by Malakay-666** [comment]
Still no luck, today I made whole new system on SD and tried to install again, still same problem. When I tried to star radicale right after installation, only set to be accessible over LAN, it was working (I saw login screen, but no users so i didn´t try to login). When I stopped radicale, it caused timeout in browser. When I started again, it was working again. So I made few changes in config file - 

htpasswd_filename = /etc/radicale/users
htpasswd_encryption = plain
[auth] type = htpasswd
hosts = 0.0.0.0:5232
[storage]type = multifilesystem
[rights] type = authenticated

and when I started again, It throw me to same error as before. I am really mad from this. I just want to sync my calendar between phone and PC. Nothing more. I spent x hours to get it work and nothing.

**Comment by CocolinoFan** [comment]
On PiOS with Radicale installed from the repository, I found that running this command fixed the issue: `sudo chown -R radicale:radicale /var/lib/radicale`

**Reply by ludo-scred** [reply]
Worked for me. I'm using Radicale debian package.
Thanks
